charts/argo-workflows/templates/controller/workflow-controller-cluster-roles.yaml

```diff
@@ -164,6 +164,19 @@ rules:
   resourceNames:
   {{/* for HTTP templates */}}
   - argo-workflows-agent-ca-certificates
+{{- if .Values.controller.rbac.secretWhitelist }}
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
+  {{- with .Values.controller.rbac.secretWhitelist }}
+  resourceNames: {{- toYaml . | nindent 4 }}
+  {{- end }}
+{{- end }}
```

Comment:
@emmayylu 
Please adopt this. 🙋 
https://github.com/argoproj/argo-helm/pull/1646#issuecomment-1316750593

Reply:
why `.Values.controller.secretWhitelist` instead of `.Values.controller.rbac.secretWhitelist`?

Reply:
Thanks for your prompt reply :) I am curious why `.Values.controller.secretWhitelist` instead of `.Values.controller.rbac.secretWhitelist`?

Reply:
@emmayylu 
ah, sorry for the lack of explanation. 😇 
I mean, instead of below

```yaml
{{- if .Values.controller.rbac.secretWhitelist }}
- apiGroups:
  - ""
  resources:
  - secrets
  verbs:
  - get
  - list
  - watch
  {{- with .Values.controller.rbac.secretWhitelist }}
  resourceNames: {{- toYaml . | nindent 4 }}
  {{- end }}
{{- end }}
```

we can set like below. 🙋 
```yaml
{{- with .Values.controller.rbac.secretWhitelist }}
- apiGroups:
  - ""
  resources:
  - secrets
  verbs:
  - get
  - list
  - watch
  resourceNames: {{- toYaml . | nindent 4 }}
{{- end }}
```

Reply:
`.Values.controller.rbac.secretWhitelist` will be nice! :)

Reply:
oh good point! Thanks! I will change it